app/src/main/java/ithaca_transit/android/cornellappdev/com/ithaca_transit/MapsActivity.java

```diff
@@ -83,16 +95,55 @@ protected void onCreate(Bundle savedInstanceState) {
         setContentView(R.layout.activity_maps);
         fusedLocationClient = LocationServices.getFusedLocationProviderClient((Activity) this);
         FragmentManager manager = this.getFragmentManager();
-        Repository.getInstance().setContext(this);
-        mController = new MapsPresenter(manager);
+        Repository.ourInstance.setContext(this);
```

Comment:
The preferred way to call the `Repository` instance is by using `private Repository rInstance = Repository.getInstance();` and then `rInstance.setContext(this)`